Documentation/proposals/app-level-api.md

```diff
@@ -19,6 +19,15 @@ aforementioned stage1 entrypoints.
 
 The proposed app-level commands are described below.
 
+## `rkt app init`
+Initializes an empty pod having no applications. This returns a single line
+containing the `pod-uuid` which can be used to perform application
```

Comment:
This sentence does not imply that the command returns immediately but only returning a single line.

Implementation-wise I would not create systemd unit files in rkt itself in the background since we do not know whether the host runs systemd. As discussed today with @lucab the easiest option would be to leave this process long-running. So another instance (like rktnetes) would have to wrap this call in a systemd unit file (which is also the current state in rktnetes).

We could also give this subcommand additionally a `--uuid-file-save=` option as in `rkt run` to save the generated pod uuid.